controllers/configentry_controller_test.go

```diff
@@ -140,6 +181,11 @@ func TestConfigEntryControllers_createsConfigEntry(t *testing.T) {
 				Address: consul.HTTPAddr,
 			})
 			req.NoError(err)
+			if c.consulPrereq != nil {
```

Comment:
Needed to add `consulPrereq` to all the tests because `service-router` (and `service-splitter`) require the protocol of the service to be set (which requires `service-defaults`)